### ISSUE TITLE
VueCompilerError: v-else/v-else-if has no adjacent v-if

### ISSUE THREAD

**Issue by marcusds** [vuejs/eslint-plugin-vue#1316]
Vue 3.0 doesn't like inline ifs with spaces between the elements. It would really handy if there was a rule to catch this and to automatically fix it. Especially for migrating from vue 2.

[X] Enforces code style (layout)
[X] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<span v-if="1">1</span> <span v-else>2</span>
<span v-if="1">1</span> <span v-else-if="3">3</span> <span v-else>2</span>
<span v-if="1">1</span> <span v-else-if="3">3</span><span v-else>2</span>
```

Valid code would be:
```vue
<span v-if="1">1</span><span v-else>2</span>
<span v-if="1">1</span><span v-else-if="3">3</span><span v-else>2</span>
<span v-if="1">1</span>
<span v-else-if="3">3</span>
<span v-else>2</span>
```

**Additional context**
Vue does throw an error, but ideally lint would catch it before it gets to that.

VueCompilerError: v-else/v-else-if has no adjacent v-if.
3  |  		<span v-if="1">1</span> <span v-else>2</span>
4  |  		<span v-if="1">1</span> <span v-else-if="3">3</span> <span v-else>2</span>
5  |  		<span v-if="1">1</span> <span v-else-if="3">3</span><span v-else>2</span>
   |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by ota-meshi** [comment]
Thank you for this issue!

However, it also looks like a bug in the Vue 3 compiler.

Can you report it in the vue-next repository?
https://github.com/vuejs/vue-next/issues

**Comment by marcusds** [comment]
I'm glad it's a bug in Vue, send like a horrible restriction.

**Comment by oswaldofreitas** [comment]
I had similar issue and opened an issue [here](https://github.com/vuejs/vue-next/issues/5210)
